app/controllers/service_calendars_controller.rb

```diff
@@ -89,13 +89,13 @@ def table
   end
 
   def merged_calendar
-    @tab          = params[:tab]
-    @review       = params[:review] == 'true'
-    @portal       = params[:portal] == 'true'
-    @admin        = @portal && @sub_service_request.present?
-    @merged       = true
-    @consolidated = false
-
+    @tab              = params[:tab]
+    @review           = params[:review] == 'true'
+    @portal           = params[:portal] == 'true'
+    @admin            = @portal && @sub_service_request.present?
+    @merged           = true
+    @consolidated     = false
+    @statuses_hidden  = []
```

Comment:
Yeah but he could instantiate the array with the params in one line, correct?. Unless I'm not seeing something.